deployments/ansible/roles/collector/README.md

```diff
@@ -4,6 +4,16 @@ Ansible role that installs Splunk OpenTelemetry Connector configured to
 collect metrics, traces and logs from Linux machines and send data to [Splunk 
 Observability Cloud](https://www.splunk.com/en_us/observability.html). 
 
+## Windows
```

Comment:
It is confusing that a Windows header exists, but not a Linux -- made me initially think this page was just for Windows. If we add for Windows we should add for Linux as well and section header generic.

Reply:
Yes, I will add Linux as well.